main.js

```diff
@@ -0,0 +1,64 @@
+// 'use strict'; 
+
+let postsEnvironment = document.querySelector('.postsEnvironment'); 
+
+async function getDate(url) {
+  const response = await fetch(url); 
+  return await response.json(); 
+}
+
+Promise.all([getDate('https://jsonplaceholder.typicode.com/posts'), 
+getDate('https://jsonplaceholder.typicode.com/users'), 
+getDate('https://jsonplaceholder.typicode.com/comments')])
+.then(result => {
+  const [postsList, usersList, commentsList] = result; 
+
+  for (let i = 0; i < postsList.length; i++) {
+    let currentPost = postsList[i]; 
+    let currentUser = usersList.find(user => user.id === currentPost.userId); 
+    let currentComments = commentsList.filter(comment => 
```

Comment:
якщо таким чином перенесиш, то записуй так
```suggestion
    const currentComments = commentsList.filter(comment => (
      comment.postId === currentPost.id
    ));
```